### ISSUE TITLE
Passing in slotProps (MUI v6) for endAdornment causes country code to not render

### ISSUE THREAD

**Issue by shadoath** [viclafouch/mui-tel-input#158]
I noticed that if I pass in slot props:

```typescript
<MuiTelInput
  label="Phone"
  value={phone}
  onChange={handleChange}
  slotProps={{
    input: {
      endAdornment: (
        <Tooltip title="Request verification code">
          <IconButton onClick={requestPhoneCode}>
            <Sms />
          </IconButton>
        </Tooltip>
      ),
    },
  }}
/>
```

Looking at the code, this is because the old `InputProps` are being used instead. I was able to get my `endAdornment` to render by using the deprecated InputProps.

Requesting that support for the new `slotProps` be added. I see that there have been recent changes to support MUI v6 (I only recently upgraded myself), so I can understand it may be planned.

Thank you!

**Comment by shadoath** [comment]
A basic attempt at a solution: #159

**Comment by viclafouch** [comment]
Hello ! 

This has been fixed in the last version (7.0.0).

Thanks !
